pkg/apis/core/types.go

```diff
@@ -1715,7 +1715,7 @@ type VolumeMount struct {
 	// Behaves similarly to SubPath but environment variable references $(VAR_NAME) are expanded using the container's environment.
 	// Defaults to "" (volume's root).
 	// SubPathExpr and SubPath are mutually exclusive.
-	// This field is beta in 1.15.
+	// This field is GA in 1.17.
```

Comment:
I believe we don't comment on GA status in the types.go

Reply:
Removed